changelog/pending/20240520--engine--guess-references-to-deeply-nested-properties-between-dependant-resources-during-import.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: engine
+  description: Guess references to (deeply nested) properties between dependant resources during import
```

Comment:
Nit: Update to mention the heuristics are extended to look at `name` and `arn` properties.

Reply:
Nice catch! updated ✅